docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/attribution/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Attribution"
+sidebar_position: 400
+hide_table_of_contents: true
+---
+
+import { DbtCongfigurationPage, JsonToTable, JsonSchemaGenerator } from '@site/src/components/JsonSchemaValidator';
+import { dump } from 'js-yaml';
+import Admonition from '@theme/Admonition';
+import DbtSchemas from "@site/docs/reusable/dbt-schemas/_index.md";
+import CodeBlock from '@theme/CodeBlock';
+import { SchemaSetter } from '@site/src/components/DbtSchemaSelector';
+
+export const printSchemaVariables = (manifestSchema, scratchSchema, derivedSchema) => {
+  return(
+    <>
+    <CodeBlock language="yaml">
+    {`models:
+  snowplow_attribution:
+    +schema: ${derivedSchema}`}
+        </CodeBlock>
+    </>
+  )
+}
+
+export const printYamlVariables = (data) => {
+  return(
+    <>
+      <h4>Project Variables:</h4>
+      <CodeBlock language="yaml">{dump({vars: {"snowplow_attribution": data}}, { flowLevel: 3 })}</CodeBlock>
+    </>
+  )
+}
+
+export const MyMdxComponent = () => {
+  const schemaName = 'dbtAttribution';
+  const versions = ['0.1.0'];
+  const label = 'dbt attribution version';
+  const output = printYamlVariables;
+    return (
+      <DbtCongfigurationPage schemaName={schemaName} versions={versions} label={label}>
+        <JsonToTable>
+            <h2 id="Variables">Package Configuration Variables</h2>
+            <p>This package utilizes a set of variables that are configured to recommended values for optimal performance of the models. Depending on your use case, you might want to override these values by adding to your <code>dbt_project.yml</code> file.</p>
+            <Admonition type="note">
+              All variables in Snowplow packages start with `snowplow__` but we have removed these in the below tables for brevity.
+            </Admonition>
+        </JsonToTable>
+        <JsonSchemaGenerator output={output}>
+            <h2 id="Generator">Config Generator</h2>
+            <p>You can use the below inputs to generate the code that you need to place into your <code>dbt_project.yml</code> file to configure the package as you require. Any values not specified will use their default values from the package.</p>
+        </JsonSchemaGenerator>
+        <DbtSchemas/>
+        <h2 id="schemas">Ouput Schemas</h2> 
```

Comment:
I think this is misspelt on all the packages config pages
```suggestion
        <h2 id="schemas">Output Schemas</h2> 
```

Reply:
I'll raise a quick PR to change this for the existing packages config pages

Reply:
lol
want to add it to https://github.com/snowplow/documentation/pull/742